contrib/system_containers/centos/Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL com.redhat.component="cri-o" \
       maintainer="Yu Qi Zhang <jzehrarnyg@gmail.com>" \
       atomic.type="system"
 
-RUN yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-container-common-candidate/x86_64/os/ && \
+RUN yum-config-manager --nogpgcheck --add-repo https://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin39/ && \
```

Comment:
can you include an additional `RUN yum-config-manager --nogpgcheck --add-repo https://buildlogs.centos.org/centos/7/virt/x86_64/container/ && \` here? This is because skopeo and the other packages you're looking for are in the Virt SIG repo and not the PaaS repo. The virt sig repo should work for all origin versions btw.

Reply:
umm, sorry I guess add the line mentioned above without the "RUN". Basically you need both those repos included.

Reply:
thanks, now the image is built correctly